float64 latitude
float64 longitude
float64 altitude
float64 radius
float32 angular_speed
uint8 is_clockwise
uint8 start_point
uint8 yaw_mode
